src/schema.types.ts

```diff
@@ -626,3 +637,23 @@ export const GraphQLNone = new GraphQLScalarType({
     return null;
   },
 });
+
+export const GraphQLEmptyObject = new GraphQLObjectType({
+  name: 'Empty',
```

Comment:
suggestion: I'd name it EmptyObject, because Empty can be also List. And might be hard to understand differences between None and Empty.

```suggestion
  name: 'EmptyObject',
```